config/robots.rb

```diff
@@ -2799,4 +2799,86 @@
         }
       }
     )
+
+    # Bioscan robots
+
+    custom_robot(
+      'beckman-lilys-96-stock-preparation',
+      name: 'Beckman LILYS-96 Stock Preparation',
+      beds: {
+        bed(9).barcode => {
+          purpose: 'LILYS-96 Stock',
+          states: ['passed'],
+          label: 'Bed 9',
+          target_state: 'passed'
+        }
+      }
+    )
+
+    custom_robot(
+      'beckman-lilys-96-stock-to-lbsn-96-lysate',
+      name: 'Beckman LILYS-96 Stock => LBSN-96 Lysate',
+      verify_robot: true,
+      beds: {
+        # TODO: confirm beds
+        bed(9).barcode => {
+          purpose: 'LILYS-96 Stock',
+          states: ['passed'],
+          label: 'Bed 9',
+          target_state: 'passed'
+        },
+        bed(14).barcode => {
+          purpose: 'LBSN-96 Lysate',
+          states: ['pending'],
+          label: 'Bed 14',
+          target_state: 'passed',
+          parent: bed(9).barcode
+        }
+      }
+    )
+
+    custom_robot(
+      'mosquito-lbsn-96-lysate-to-lbsn-384-pcr-1',
+      name: 'Mosquito LBSN-96 Lysate => LBSN-384 PCR 1',
+      beds: {
+        bed(1).barcode => {
+          purpose: 'LBSN-96 Lysate',
+          states: ['passed'],
+          child: bed(5).barcode,
+          label: 'Bed 1'
+        },
+        bed(2).barcode => {
+          purpose: 'LBSN-96 Lysate',
+          states: ['passed'],
+          child: bed(5).barcode,
+          label: 'Bed 2'
+        },
+        bed(3).barcode => {
+          purpose: 'LBSN-96 Lysate',
+          states: ['passed'],
+          child: bed(5).barcode,
+          label: 'Bed 3'
+        },
+        bed(4).barcode => {
+          purpose: 'LBSN-96 Lysate',
+          states: ['passed'],
+          child: bed(5).barcode,
+          label: 'Bed 4'
+        },
+        # TODO: does the order of the parents here define the quadranting behaviour?
```

Comment:
Should create a backlog item if this TODO is still needed.

Reply:
remove comment, it does